scripts/build.py

```diff
@@ -0,0 +1,28 @@
+import os
+import subprocess
+import warnings
+
+
+ROOT = os.path.join(os.path.dirname(__file__), "..")
+
+
+def _exec(cmd, default):
+    try:
+        return subprocess.check_output(
+            cmd.split(" "), encoding="utf-8", cwd=ROOT
+        ).strip()
+    except subprocess.CalledProcessError:
+        warnings.warn("out of tree; milabench could not retrieve version info")
+        return default
```

Comment:
This could be imported from milabench.metadata to improve code reuse (also this correctly sets cwd for the git command, whereas the code in milabench.metadata does not).

Reply:
Importing it from milabench would cause a circular dependency no ?
Since this is called before milabench is installed

Reply:
It might not if the files are already where Python can see them when the script is run. Either way, let's not spend too much time on this, let's only make sure both _exec functions work the same (wrt cwd for example).

Reply:
I tried and it was not working.
But I think I found an alternative by creating a vcs.py file that does not dependent on milabench I am able to reuse the code for both build time and metadata retrieval